h2o-core/src/main/java/water/ExternalFrameHandler.java

```diff
@@ -70,7 +71,7 @@
     /**
      * Method which receives the {@link SocketChannel} and {@link AutoBuffer} and dispatches the request for further processing
```

Comment:
Minor thing - the link in the comment should be changed as well to `ByteChannel`